src/main/java/service/impl/WriteToStorageImpl.java

```diff
@@ -0,0 +1,37 @@
+package service.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import exception.CheckSplitLine;
+import exception.CheckSplitLineImpl;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import model.Fruit;
+import service.GetFruitFromFile;
+import service.GetValueForFruit;
+import service.WriteToStorage;
+
+public class WriteToStorageImpl implements WriteToStorage {
+    private static final int FRUIT_INDEX = 1;
+
+    private final FruitDao fruitDao = new FruitDaoImpl();
+    private final GetFruitFromFile getFruitFromFile = new GetFruitFromFileImpl();
+    private final GetValueForFruit getValueForFruit = new GetValueForFruitImpl();
+    private final CheckSplitLine checkSplitLine = new CheckSplitLineImpl();
```

Comment:
all these injections shouldn't be there, this violates SRP. This class should only be concerns with writing given string to a given path